File: src/main/java/um/haberes/report/model/haberes/core/ItemDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ItemDto {

    private Long itemId;

    private Long legajoId;

    private int anho = 0;

    private int mes = 0;

    private Integer codigoId;

    private String codigoNombre = "";

    private BigDecimal importe = BigDecimal.ZERO;

    private PersonaDto persona;

    private CodigoDto codigo;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/ItemDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class ItemDto(

    var itemId: Long? = null,
    var legajoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var codigoId: Int? = null,
    var codigoNombre: String = "",
    var importe: BigDecimal = BigDecimal.ZERO,
    var persona: PersonaDto? = null,
    var codigo: CodigoDto? = null,

    )
